docs/blog/pydantic-v2-alpha.md

```diff
@@ -0,0 +1,182 @@
+
+This will be the first of several upcoming alpha releases. As you evaluate our changes and enhancements, 
+we encourage you to share your feedback with us.
+
+Please let us know:
+
+* If you don't like the changes, so we can make sure Pydantic remains a library you enjoy using.
+* If this breaks your usage of Pydantic so we can fix it, or at least describe a migration path.
+
+Thank you for your support, and we look forward to your feedback.
+
+---
+
+## Headlines
+
+Here are some of the most interesting new features in the current Pydantic V2 alpha release. 
+For background on plans behind these features, see the earlier [Pydantic V2 Plan](/blog/pydantic-V2/) blog post.
+
+The biggest change to Pydantic V2 is [`pydantic-core`](https://github.com/pydantic/pydantic-core) &mdash;
+all validation logic has been rewritten in Rust and moved to a separate package, `pydantic-core`.
+This has a number of big advantages:
+
+* **Performance** - Pydantic V2 is 5-50x faster than Pydantic V1.
+* **Safety** - Pydantic V2 should have far fewer bugs long term.
+* **Maintainability** - we're hoping that, long term, V2 is easier to extend without the risk of introducing bugs or breaking changes.
+
+With the use of `pydantic-core`, the majority of the logic in the Pydantic library is dedicated to generating
+"pydantic core schema" &mdash; the schema used define the behaviour of `pydantic-core` validators and serializers.  
+
+### Ready for experimentation
+
+* **BaseModel** - the core of validation in Pydantic V1 remains, albeit with new method names.
+* **dataclasses** - Pydantic dataclasses are improved and ready to test.
+* **Serialization** - dumping/serialization/marshalling is significantly more flexible, and ready to test.
+* **strict mode** - one of the biggest additions in Pydantic V2 is strict mode, which is ready to test.
+* **JSON Schema** - generation of JSON Schema is much improved and ready to test.
+* **Generic Models** - are much improved and ready to test.
+* **Recursive Models** - and validation of recursive data structures is much improved and ready to test.
+* **Custom Types** - custom types have a new interface and are ready to test.
+* **Custom Field Modifiers** - used via `Annotated[]` are working and in use in Pydantic itself. 
+* **Validation without a BaseModel** - the new `AnalyzedType` class allows validation without the need for a `BaseModel` class, and it's ready to test.
+* **TypedDict** - we now have full support for `TypedDict` via `AnalyzedType`, it's ready to test.
+
+### Still under construction
+
+* **Documentation** - we're working hard on full documentation for V2, but it's not ready yet.
+* **Conversion Table** - a big addition to the documentation will be a conversion table showing how types are coerced, this is a WIP.
+* **BaseSettings** - `BaseSettings` will move to a separate `pydantic-settings` package, it's not yet ready to test.
```

Comment:
@samuelcolvin we probably need to give people a suggestion of what to do, since `BaseSettings` isn't in pydantic v2. If they are using it in their codebase, I think they won't be able to test out the alpha.